src/DebugConfigurationProvider.ts

```diff
@@ -307,6 +309,8 @@ export class BrightScriptDebugConfigurationProvider implements DebugConfiguratio
         config.disableScreenSaver = config.disableScreenSaver === false ? false : this.configDefaults.disableScreenSaver;
         config.retainStagingFolder ??= this.configDefaults.retainStagingFolder;
         config.enableVariablesPanel = 'enableVariablesPanel' in config ? config.enableVariablesPanel : this.configDefaults.enableVariablesPanel;
+        config.autoResolveVirtualVariables = config.autoResolveVirtualVariables === true ? true : this.configDefaults.autoResolveVirtualVariables;
+        config.enhanceREPLCompletions = config.enhanceREPLCompletions === true ? true : this.configDefaults.enhanceREPLCompletions;
```

Comment:
All of these failing checks can be resolved now by running `npm install roku-debug@latest` and committing the package and package-lock